i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+from typing import Callable
+import torch
+from torch import nn
+
+
+class ConformerPositionwiseFeedForwardV1(nn.Module):
+    """
+    Conformer feedforward module
+    """
+
+    def __init__(
+        self,
+        input_dim: int,
```

Comment:
Please change the input to take some kind of ConformerFeedForwardConfig (c.f. #8 for inspiration) and also define that Config in this file, as agreed on in the last meeting.

Reply:
The inputs are defined as Config class now